app/src/main/java/bubbleheads/buy_cook/CategoriesAdapter.java

```diff
@@ -0,0 +1,58 @@
+package bubbleheads.buy_cook;
+
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentStatePagerAdapter;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CategoriesAdapter extends RecyclerView.Adapter<CategoriesAdapter.MyViewHolder>{
+    private CategoriesFragment categoriesFragment;
+    private ArrayList<Category> categoriesList = new ArrayList<>();
```

Comment:
do you actually need this collection? I mean you can access enum members directly from Category.values() with no need to copy it here